ports/mimxrt/Makefile

```diff
@@ -162,6 +163,13 @@ SRC_HAL_IMX_C += \
 	$(MCU_DIR)/drivers/fsl_romapi.c
 endif
 
+# If not empty, then it is 10xx.
+ifneq ($(findstring MIMXRT10, $(MCU_SERIES)),)
+BOOTLOADER_SIZE := 0x6000C000
```

Comment:
Maybe this variable would be better called `APPLICATION_ADDR`?

Reply:
Sure I can change that, because that's what it is.

Reply:
Updated.